app/code/Magento/ImportExport/Test/Unit/Model/Import/Source/FileParser/FakeParserFactory.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\ImportExport\Test\Unit\Model\Import\Source\FileParser;
+
+use Magento\ImportExport\Model\Import\Source\FileParser\ParserFactoryInterface;
+use Magento\ImportExport\Model\Import\Source\FileParser\UnsupportedPathException;
+
+class FakeParserFactory implements ParserFactoryInterface
```

Comment:
We have so many Fake objects here, and we need to update their code updating code of the real objects as well 

Currently, we don't use this approach for Unit testing. And definitely it  needs separate approval from AC 

Because for now we introduced additional entities to maintain. But why do we need that if we already have Integration tests?